### ISSUE TITLE
AuditApiAdapter (.NET Framework version) is throwing System.NullReferenceException

### ISSUE THREAD

**Issue by hainguyenthanh** [thepirat000/Audit.NET#146]
**Describe the bug**
It seems that there's a missing null-conditional access operator in the `IsActionIgnored()` in the AuditApiAdapter.cs file, which causes a `NullReferenceException` exception. The line of code causing the issue is

`var actionIgnored = actionDescriptor.MethodInfo.GetCustomAttribute<AuditIgnoreAttribute>(true);`

Comparing to the code that checks if the controller is ignored, the null-conditional access operator was used.

`var controllerIgnored = actionDescriptor?.MethodInfo.DeclaringType.GetTypeInfo().GetCustomAttribute<AuditIgnoreAttribute>(true);`

**To Reproduce**
Creating an ASP.NET Web API project that references and uses the Audit.WebAPI package, and decorate a controller or an action method with the `AuditApiAttribute`.

**Expected behavior**
`NullReferenceException` should not be thrown.

**Libraries (specify the Audit.NET extensions being used including version):**
 - Audit.WebApi: 12.2.2

**Additional context**
AuditApiAdapter.Core.cs has a check for `null` in its code.

**Comment by hainguyenthanh** [comment]
Hi @thepirat000,

I think that the .NET Framework version of `IsActionIgnored()` method may be enhanced by eliminating the casting to `ReflectedHttpActionDescriptor`. Please take a look.

```c#
public bool IsActionIgnored(HttpActionContext actionContext)
{
    var actionDescriptor = actionContext.ActionDescriptor;

    var controllerIgnored = actionDescriptor.ControllerDescriptor.GetCustomAttributes<AuditIgnoreAttribute>(true).Any();
    if (controllerIgnored)
    {
        return true;
    }

    var actionIgnored = actionDescriptor.GetCustomAttributes<AuditIgnoreAttribute>(true).Any();
    if (actionIgnored)
    {
        return true;
    }

    return false;
}
```

Regards,

**Comment by thepirat000** [comment]
Thanks!

This fix was added on version 12.3.1, please upgrade your references and reply to this if you're still having problems.
